docs/ref/models/expressions.txt

```diff
@@ -328,6 +328,15 @@ instantiating the model field as any arguments relating to data validation
 (``max_length``, ``max_digits``, etc.) will not be enforced on the expression's
 output value.
 
+Conditional expressions
+-----------------------
+
+.. versionadded:: 1.8
+
+Conditional expressions allow you to use :keyword:`if` ... :keyword:`elif` ...
+:keyword:`else` logic in queries. Django supports SQL simple and searched
```

Comment:
simple case expressions are no longer supported since removal of SimpleCase right?